app/src/main/java/org/feature/fox/coffee_counter/ui/authentication/AuthenticationViewModel.kt

```diff
@@ -76,8 +78,20 @@ class AuthenticationViewModel @Inject constructor(
         if (response.data == null) {
             toastMessage.value = response.message ?: resource.getString(R.string.unknown_error)
             return
+        } else {
+            userRepository.insertUser(
+                User(
+                    id = idState.value.text,
+                    name = nameState.value.text,
+                    false,
+                    password = BCrypt.withDefaults()
+                        .hashToString(12, passwordState.value.text.toCharArray())
+                )
+            )
         }
 
+
+
```

Comment:
Maximum one empty line between code inside a function to seperate logical blocks is the as far as i know the current standard.